site/gdocs/components/Image.tsx

```diff
@@ -52,6 +53,7 @@ const containerSizes: Record<ImageParentContainer, string> = {
     ["datapage"]: gridSpan6,
     ["full-width"]: "100vw",
     ["key-insight"]: gridSpan5,
+    ["author-byline"]: "48px",
```

Comment:
I wasn't able to make my browser use this size. @ikesau do you know why that might be?

Reply:
The dev server doesn't use 'em because we only resize the images during baking.

https://github.com/owid/owid-grapher/blob/7527f17767410896b567916e3ef431b703f314c2/site/gdocs/components/Image.tsx#L116

It's working correctly on staging 👍

Reply:
Well, my browser won't use the 48px even on staging. It uses `_100.png` even when 48px is available, and I'm not sure if the 48px will ever be used then. Does your browser pick the 48px?

Reply:
Ah that's a pixel density thing probably. Try testing it with your browser's Responsive Design Mode and set the DPR to 1. 🙂

Reply:
Nice, that works. I somehow didn't realize browser tools allow you to set the DPR.